packages/mdc-dom/package.json

```diff
@@ -4,6 +4,9 @@
   "version": "0.41.0",
   "license": "MIT",
   "main": "dist/mdc.dom.js",
+  "module": "index.js",
+  "sideEffects": false,
+  "types": "dist/mdc.dom.d.ts",
```

Comment:
Why do other packages depend on `tslib` while this one doesn't?

Reply:
Probably because it doesn't use any TS syntax requiring helper functions. I think we should probably add tslib to all packages that contain TS though, because this is very likely to silently break the moment we unwittingly add something that needs it.

Reply:
The `lint:imports` check will actually catch this. Otherwise we'll add a package that isn't being used.

Reply:
You're referring to running `lint:imports` specifically on the generated JS? Weren't we thinking of updating that to run on the `ts` itself? Otherwise `lint:imports` is a no-op without running `npm run build:esmodules` first (which for example our lint task on Travis CI won't do). But if we convert it to TS it will no longer catch this.

Reply:
After talking with Andy this morning, I've added tslib to the animation and dom packages, which are the only individual packages containing TS that didn't already have it. I think it's safe to leave it out of material-components-web since we only ever add to the existing code without adding entirely new structures. This way we can eventually convert the import check script to look at the TS files along with what's in #4449.